style/client.css

```diff
@@ -2303,6 +2306,7 @@ a.ilink:hover {
 }
 .setchart .setcell-item input {
 	width: 114px;
+	margin-right: 1px;
```

Comment:
It was 1px because I was experimenting with changing margins in various places to see what worked and found that 1px lets me zoom to 67%; to zoom to 50% I'd need to set both this and the left margin on the ability to zero.